OpenRA.Mods.Common/Widgets/Logic/ServerBrowserLogic.cs

```diff
@@ -135,14 +135,7 @@ public ServerBrowserLogic(Widget widget, Action onStart, Action onExit, string d
 				showIncompatibleCheckbox.OnClick = () => { showIncompatible ^= true; RefreshServerList(); };
 			}
 
-			try
-			{
-				Game.LoadWidget(null, "SERVERBROWSER_IRC", panel.Get("IRC_ROOT"), new WidgetArgs());
-			}
-			catch
-			{
-				Log.Write("debug", "Failed to load server browser IRC chrome layout");
-			}
+			Game.LoadWidget(null, "GLOBALCHAT_PANEL", panel.Get("GLOBALCHAT_ROOT"), new WidgetArgs());
```

Comment:
I don't think we should be making any widget mandatory, and we shouldn't force (our!) IRC on thirdparty mods.